src/modclub_assets/src/utils/gtm.ts

```diff
@@ -1,12 +1,40 @@
 import { hideStringWithStars } from "./util";
 
 export enum GTMEvent {
-  UserCreatedProfile = "userCreatedProfile",
-  UserCreateProfileError = "userCreateProfileError",
-  LaunchApp = "launchApp",
-  UserPohChallenge = "userPohChallenge",
-  AccountTransaction = "accountTransaction",
-  HumanVerification = "humanVerification",
+  // --- User create profile event name ---
```

Comment:
just discuss with you: the other way is to build a nested structure:
```
enum GTMEvent {
Transaction = {
    EventName: "gtm_transaction",
    DepositEventType: "deposit",
    WithdrawEventType: "withdraw",
    StakeEventType: "stake",
    UnStakeEventType: "unstake",
    ClaimEventType: "claim",
  },
  ....
}```

Reply:
@xwdeng so can you provide us with advantages( against current ) list for such approach?

Reply:
Better readability since it is a nested structure, in my personal opinion.